checks/apps/pytorch/nvcr_utility.py

```diff
@@ -0,0 +1,28 @@
+
+import requests
+import re
+from packaging.version import Version
+
+
+def latest_nvidia_image_tags(image_name:str):
```

Comment:
You can extract his even further up the directory hierarchy, so that it can be used for other non pytorch checks. So it can be in the same depth as `checks` like a new `utility` directory

Reply:
Right! I'll move under the existing chekcs/tools/ folder